onnxruntime/core/providers/webgpu/program.cc

```diff
@@ -120,6 +123,8 @@ int NumberOfComponents(ProgramVariableDataType type) {
     case ProgramVariableDataType::Int64:
     case ProgramVariableDataType::Uint64:
     case ProgramVariableDataType::Float16:
+    case ProgramVariableDataType::Uint4x8:
+    case ProgramVariableDataType::Int4x8:
```

Comment:
Regarding the number of components, how does ORT defines the shape of a INT4 tensor?

say a simple exmaple, we have a 1-D INT4 tensor representing 1024 INT4 numbers. is the tensor shape [1024] or [512] (or other)?

Reply:
[1024] in onnx

Reply:
then to make the definition of "number of components" consistent, it should be 8 instead of 1 here.

Reply:
fixed.